### ISSUE TITLE
Last update does not start (1.38.0, 54087a3970cef9dcbbaed62de814d75ad9de59649285e495cf56d1e5512071a4)

### ISSUE THREAD

**Issue by dominikzogg** [flathub/com.mongodb.Compass#25]
```
[3 zypak-helper] exec failed for /app/main/MongoDB Compass: No such file or directory (errno 2)
```

Downgrade to the last working version at the creation of the issue:

sudo flatpak update --commit=370cd6dbf89eb1feb9c9bbcce8f0364dd9f606601e97b4ee60b65208a030ff37 com.mongodb.Compass

**Comment by dominikzogg** [comment]
https://github.com/flathub/com.mongodb.Compass/pull/29

**Comment by dominikzogg** [comment]
fixed by #29
